doc/examples_index.rst

```diff
@@ -145,13 +145,13 @@ Contextual enhancement
 Fiber tracking
 --------------
 
-- :ref:`example_introduction_to_basic_tracking`
```

Comment:
At the `Quick Start` section can you replace `example_tracking_quick_start` by `example_introduction_to_basic_tracking`

Reply:
done. example_introduction_to_basic_tracking was renamed "example_tracking_introduction_eudx"